lib/Froxlor/Cron/Dns/DnsBase.php

```diff
@@ -181,13 +181,28 @@ public function reloadDaemon()
 	public function writeDKIMconfigs()
 	{
 		if (Settings::Get('dkim.use_dkim') == '1') {
+			shell_exec('rm -rf ' . escapeshellarg(\Froxlor\FileDir::makeCorrectDir(Settings::Get('dkim.dkim_dkimkeys'))) . 'dkim_*');
+
+			shell_exec('rm -rf ' . \Froxlor\FileDir::makeCorrectFile(Settings::Get('dkim.dkim_prefix') . '/signing.table'));
+			shell_exec('rm -rf ' . \Froxlor\FileDir::makeCorrectFile(Settings::Get('dkim.dkim_prefix') . '/key.table'));
+			shell_exec('rm -rf ' . \Froxlor\FileDir::makeCorrectFile(Settings::Get('dkim.dkim_prefix') . '/66-froxlor-dkim'));
+
```

Comment:
1) please use \Froxlor\FileDir::safe_exec()
2) Why use parameter for `recursive` if you just deleting files? `rm -f` should be more than enough